test/test.jl

```diff
@@ -75,11 +75,14 @@ tse_str = sprint(show, Test.TestSetException(1,2,3,4))
 @test Test.finish(Test.FallbackTestSet()) !== nothing
 
 OLD_STDOUT = STDOUT
+OLD_STDERR = STDERR
 catch_out = IOStream("")
-rd, wr = redirect_stdout()
+catch_err = IOStream("")
+rde, wre = redirect_stderr()
+rdo, wro = redirect_stdout()
 
 # Check that the fallback test set throws immediately
-@test_throws ErrorException (@test 1 == 2)
+#@test_throws ErrorException (@test 1 == 2)
```

Comment:
will be important to continue testing that we don't break this, otherwise something might regress which could give green travis statuses even when tests fail